apps/web/vibes/soul/docs/alert.mdx

```diff
@@ -12,21 +12,39 @@ features:
 
 ## Usage
 
-<CodeBlock lang="bash">{`import { Alert } from '@/vibes/soul/primitives/alert'
-import { useState } from 'react'`}</CodeBlock>
+<CodeBlock lang="ts">{`import { Alert } from '@/vibes/soul/primitives/alert'`}</CodeBlock>
 
-<CodeBlock lang="bash">{`const [showAlert, setShowAlert] = useState(false)`}</CodeBlock>
-
-<CodeBlock lang="bash">{`<Alert variant="success" message="1 item added to your cart" showAlert={showAlert} onClose={setShowAlert}/>`}</CodeBlock>
+<CodeBlock lang="ts">{`<Alert message="This is a success alert" variant="success" />`}</CodeBlock>
 
 ## API Reference
 
-| Prop                                                                                   | Type                                                              | Default     |
-| -------------------------------------------------------------------------------------- | ----------------------------------------------------------------- | ----------- |
-| `variant` <Tooltip content='The style variant of the alert, defines color and icon' /> | `enum` <Tooltip content={`"success" "error" "warning" "info"`} /> | `"success"` |
-| `message` <Tooltip content='Alert message displayed' />                                | `string`                                                          |             |
-| `showAlert`                                                                            | `boolean`                                                         |             |
-| `onClose`                                                                              | `(showAlert: boolean) => void`                                    |             |
+### `AlertProps`
+
+| name                                                                                    | type                                          | default     |
+| --------------------------------------------------------------------------------------- | --------------------------------------------- | ----------- |
+| `variant*` <Tooltip content='The style variant of the alert, defines color and icon' /> | `"success" \| "warning" \| "error" \| "info"` |             |
+| `message*` <Tooltip content='Alert message displayed' />                                | `ReactNode`                                   |             |
+| `description`                                                                           | `string`                                      |             |
+| `dismissLabel`                                                                          | `string`                                      | `"Dismiss"` |
+| `action`                                                                                | `{ label: string; onClick: () => void; }`     |             |
+| `onDismiss`                                                                             | `() => void`                                  |             |
+
+### CSS Variables
```

Comment:
I feel like CSS variables make sense to include as a section under API Reference. I feel like it provides enough context. What do you think @hunterbecton ?